.gitignore

```diff
@@ -16,3 +16,5 @@ tmp/
 # Mac OSX Finder files.
 **/.DS_Store
 .DS_Store
+
+yarn.lock
```

Comment:
In fact, i use another npm registry for fast installation. And yarn hard code the registry url in the yarn.lock. That may be incompatible with other contributors. To keep the working directory clean, i want to ignore it.

An other concern is there is no `yarn.lock` committed before. I'm not sure if you guys are comfort with it. :smile: